guides/dev/tests/README.md

```diff
@@ -140,6 +140,47 @@ Please note that some CKEditor plugins are needed for reasons that might not be
 
 If the editor behaves differently when testing and during development, try to add all plugins you use during the development (you can find such list in the [`config.js`](https://github.com/ckeditor/ckeditor-dev/blob/master/config.js) file) and then remove redundant ones. Please note that adding all existing CKEditor plugins might not be a good solution since, for example, the [BBCode plugin](http://ckeditor.com/addon/bbcode) will strip HTML in your output.
 
+### Tagging Tests
+
+Each test should be properly tagged, so it is easy to group and filter them while running. Tags should be added using `bender-tags` meta comment, like:
+
+    /* bender-tags: editor */
+    // Rest of the file...
+
+In case of unit tests `bender-tags` meta comment should be placed in `*.js` file and in `*.md` file for manual tests.
+
+**Common tags used in tests**:
+
+* bug/feature - test covers bug fix or a new feature. If test suit contains more general tests this tag should be omitted,
+* &lt;CKEditor version&gt; - CKEditor target version,
+* &lt;number&gt; - reference to GitHub issue,
```

Comment:
Not sure about _optional_ part, if you have any idea how it may be marked to be more readable than what I did now (https://github.com/ckeditor/ckeditor-docs/pull/104/commits/c2c6126355ab45056fea73818be3dc6f61e5db4d#diff-4b8d5c57f14797f313f3adfb81d70582R154) I am open for new ideas.